checks/ulimits_test.go

```diff
@@ -0,0 +1,123 @@
+package checks
+
+import (
+	"syscall"
+	"testing"
+)
+
+func TestEveryType(t *testing.T) {
+
```

Comment:
empty new line at top of functions is inconsistent with rest of the code base.

Reply:
removed